### DISCUSSION TITLE
How could we embed a terminal (bash, Powershell) in an AvaloniaUI

### DISCUSSION THREAD

**Discussion by zaminda** [AvaloniaUI/Avalonia#9016] [Q&A]
I'd like to embed a terminal in the UI. This could be something similar to the terminal window opening in VSCode (screenshot below). Would it be possible to do something like this? Any help would be awesome! 

Thank you very much 🙏🏽

<img width="710" alt="image" src="https://user-images.githubusercontent.com/73978236/192123526-04cd5899-1788-42e4-a324-e87b7dcc6867.png">

**Comment by kekekeks** [comment]
https://github.com/VitalElement/AvalonStudio.TerminalEmulator - not sure if it's updated to the last Avalonia version, but it used to work quite well
